emperor/support_files/js/controller.js

```diff
@@ -217,6 +220,11 @@ define([
    *
    **/
   EmperorController.prototype.render = function() {
+    if (!_.some(this.sceneViews, function(sv) {
```

Comment:
I see what you were talking about before, what about something like this:


```javascript

var shouldRender = _.some(this.sceneViews, function(sv) {
    return sv.checkUpdate();
});

if(shouldRender){
  // rendering shenanigans 
}

```

This way we get rid of the `return` and avoid having the `if` spanning multiple lines.